ledger/cow.go

```diff
@@ -285,3 +290,34 @@ func (cb *roundCowState) commitToParent() {
 func (cb *roundCowState) modifiedAccounts() []basics.Address {
 	return cb.mods.Accts.ModifiedAccounts()
 }
+
+// CalculateTotals calculates the totals given the changes in the StateDelta.
+// these changes allow the validator to validate that the totals still align with the
+// expected values. ( i.e. total amount of algos in the system should remain consistent )
+func (cb *roundCowState) CalculateTotals() error {
+	// this method applies only for the top level roundCowState
+	if cb.commitParent != nil {
+		return nil
+	}
```

Comment:
This test isn't required on its own.. but if this method would be called on a child cow state, it could get wrong results, since the `lookup` is **required** to bring the `AccountData` from the previous round.

Reply:
Maybe should return an error here? so the caller knows is doing something wrong.